loader.go

```diff
@@ -59,104 +60,115 @@ func logger() *zap.Logger {
 
 // Start docker loader
 func (dockerLoader *DockerLoader) Start() error {
-	if !dockerLoader.initialized {
-		dockerLoader.initialized = true
-		log := logger()
-
-		dockerClients := []docker.Client{}
-		for i, dockerSocket := range dockerLoader.options.DockerSockets {
-			// cf https://github.com/docker/go-docker/blob/master/client.go
-			// setenv to use NewEnvClient
-			// or manually
+	if dockerLoader.initialized {
+		return nil
+	}
 
-			os.Setenv("DOCKER_HOST", dockerSocket)
+	dockerLoader.initialized = true
+	log := logger()
 
-			if len(dockerLoader.options.DockerCertsPath) >= i+1 && dockerLoader.options.DockerCertsPath[i] != "" {
-				os.Setenv("DOCKER_CERT_PATH", dockerLoader.options.DockerCertsPath[i])
-			} else {
-				os.Unsetenv("DOCKER_CERT_PATH")
-			}
+	if envFile := dockerLoader.options.EnvFile; envFile != "" {
+		if err := godotenv.Load(dockerLoader.options.EnvFile); err != nil {
+			log.Error("Load variables from environment file failed", zap.Error(err), zap.String("envFile", dockerLoader.options.EnvFile))
+			return err
+		}
+		log.Info("environment file=%s loaded", zap.String("envFile", dockerLoader.options.EnvFile))
```

Comment:
Can you please double-check what's in your logs? I wonder if %s is doing anything, given the envFile is passed as a structured field.
If %s is not being replaced, let's just remove `=%s`?

Reply:
I double-checked, this is the output:
```
{"level":"info","ts":1701208881.6625867,"logger":"docker-proxy","msg":"environment file=%s loaded","envFile":"/etc/caddy/env"}
```
Can you please remove `=%s`?